cmd/sync/sync.go

```diff
@@ -12,9 +12,10 @@ import (
 
 // syncCmd represents the config command
 var syncCmd = &cobra.Command{
-	Use:   "sync [OPTIONS] [ARG...]",
-	Short: "Sync the backing git repo of liferay cli",
-	Args:  cobra.NoArgs,
+	Use:     "sync [OPTIONS] [ARG...]",
+	Short:   "Sync the backing git repo of liferay cli",
+	Aliases: []string{"s"},

```

Comment:
Will this get us into trouble in the future? If we have another command that might want to be shortended to `s`?

Reply:
perhaps. it may have been a little frivolous so i can remove it.

Reply:
its a 0.x so we can always change aliases in the future.  we can leave it i think.